examples/basic_tutorial_BERT.ipynb

```diff
@@ -0,0 +1,1198 @@
+{
```

Comment:
<p>here you can simply use</p><pre class="ql-syntax" spellcheck="false">from gdeep.utility import DEVICE
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-deep/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Thanks! It is a more giotto-way to do this. In general, any other comments on how to make the notebook more giotto-like are  highly appreciated.